packages/cli/src/components/ProjectStatus/ClientProjectStatus.tsx

```diff
@@ -92,9 +92,9 @@ export default function ClientProjectStatus({
       },
     )
 
-    const childProcess = spawnChildProcess(
+    const childProcess = spawnNpmExec(
```

Comment:
Can we make a seperate `spawnPheroClient` and `spawnPheroServer` function instead of a generic `spawnNpmExec`? This way we prevent usage of `spawnNpmExec` with an "executable" option from user input in the future :)